apps/theming-designer/src/components/FabricPalette.tsx

```diff
@@ -0,0 +1,142 @@
+import * as React from 'react';
+import { Card } from '@uifabric/react-cards';
+import { FabricSlots, IThemeRules } from 'office-ui-fabric-react/lib/ThemeGenerator';
+import { Stack } from '../../../../packages/office-ui-fabric-react/lib/index';
+import { mergeStyles } from '@uifabric/merge-styles';
+
+export interface IFabricPaletteProps {
+  themeRules: IThemeRules | undefined;
+}
+
+const tableClassName = mergeStyles({
+  selectors: {
+    th: {
+      padding: 10,
+      textAlign: 'left'
+    },
+    td: {
+      padding: 10,
+      textAlign: 'left'
+    }
+  }
+});
+
+const slotClassName = mergeStyles({
+  display: 'flex',
+  alignItems: 'center',
+  overflow: 'auto'
+});
+
+const fabricPaletteColorBox = mergeStyles({
+  width: 15,
+  height: 15,
+  display: 'inline-block',
+  left: 5,
+  top: 5,
+  border: '1px solid black',
+  flexShrink: 0
+});
+
+export const FabricPalette: React.StatelessComponent<IFabricPaletteProps> = (props: IFabricPaletteProps) => {
+  const fabricSlotWidget = (fabricSlot: FabricSlots): JSX.Element => {
+    const slotRule = props.themeRules![FabricSlots[fabricSlot]];
+    return (
+      <div key={slotRule.name} className={slotClassName}>
+        <Stack horizontal gap={5}>
+          <div
+            key={slotRule.name}
+            className={fabricPaletteColorBox}
+            style={{ backgroundColor: slotRule.color!.str }}
+            // onClick={} no colorpicker on click for now but in the future, possible.
+          />
+          <div>{slotRule.name}</div>
+          {/* {!slotRule.isCustomized ? <div>Inherits from: {slotRule.inherits!.name}</div> : <div>Customized</div>} */}
+        </Stack>
+      </div>
+    );
+  };
+
+  return (
+    <Card styles={{ root: { width: '800px', height: 'auto', display: 'flex' } }}>
```

Comment:
Is there a UI treatment for this display on mobile devices? Might not want to hardcode the width of this display.

Reply:
+1 on this, maybe use `minWidth` and `maxWidth` tokens instead of `width`?

Reply:
The designers don't really have a mobile story for this app at the moment. I used minWidth & maxWidth on all the cards to get started in this direction though.